File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String str[] = s.split("");
        String answer = "";
        String temp = "";
        boolean flag = false;
        for(int i = 0;i < str.length;i++) {
            if(str[i].equals(" ")) {
                if(temp.equals("")) answer += str[i];
                else {
                    temp += str[i];
                    answer += temp;
                    flag = false;
                    temp = "";
                }
                continue;
            } 
            if(!flag) {
                flag = true;
                temp += str[i].toUpperCase();
            }
            else temp += str[i].toLowerCase();
        }
        answer += temp;
        return answer;
    }
}

File: 프로그래머스/Lv.2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    
    const str = s.split(" ");
    
    for(let i = 0;i < str.length;i++){
        const word = str[i];
        answer += word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
        
        if(i !== str.length - 1) answer += " ";
    }
    
    return answer;
}